qa/sc_cert_key_rotation_across_epoch.py

```diff
@@ -17,7 +17,7 @@
 from httpCalls.submitter.getCertifiersKeys import http_get_certifiers_keys
 from httpCalls.submitter.getKeyRotationProof import http_get_key_rotation_proof
 from SidechainTestFramework.secure_enclave_http_api_server import SecureEnclaveApiServer
-from httpCalls.submitter.getSchnorrPublicKeyHash import http_get_schnorr_public_key_hash
+from httpCalls.submitter.getKeyRotationMessageToSign import http_get_key_rotation_message_to_sign
```

Comment:
We need to extend the test with one more case
Look at the description:
`- Create a CertificateKeyRotationTransaction (T3) to change the signer key 0 (SK0 -> SK3)`
Similar to that we need to create Tx T4 with valid key transition, but invalid message to sign, because of the "old" withdrawal epoch number. Same as with T3, we need to check that for the first block of the new epoch - tx is invalid.